build.xml

```diff
@@ -42,12 +44,35 @@
 	<target name="java-build-dir-exists">
 		<mkdir dir="target/asciidocextensions" />
 	</target>
+	<target name="validation-api-dir-exists">
+		<delete dir="${validation-api-source-dir}/javax/" />
+		<mkdir dir="${validation-api-source-dir}" />
+	</target>
 
+	<!-- we update the dependency every time we build as it might have changed on disk -->
+	<target name="detect-validation-api-in-m2">
+		<available file="${user.home}/.m2/repository/javax/validation/validation-api/${bv.version}/validation-api-${bv.version}-sources.jar" property="validation-api-in-m2"></available>
```

Comment:
I'd extract a var at the top of the file for the repo location so one can easily adapt it if needed.